test/object.spec.js

```diff
@@ -439,4 +440,31 @@ describe('object', function() {
 
   });
 
+
+  describe('get', function() {
+
+    it('should return object property', function() {
+      expect(get({}, ['a'])).to.equal(undefined);
+      expect(get({}, ['a'], 'FOO')).to.equal('FOO');
+
+      expect(get({ a: 0 }, ['a'])).to.equal(0);
+      expect(get({ a: 0 }, ['a'], 1)).to.equal(0);
+
+      expect(get({ a: { b: 0 } }, ['a', 'b'])).to.equal(0);
+      expect(get({ a: { } }, ['a', 'b'], 1)).to.equal(1);
+    });
```

Comment:
Do we want it to be error-proof?

```suggestion
      expect(get({ a: null }, ['a', 'b'], 1)).to.equal(1);
    });
```

Reply:
I'd say yes. I don't expect any errors to be thrown by this function.